FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -1,99 +1,118 @@
+        }
+        
+        private void CallActionsWithVectors(Vector[] vectors)
         {
-            Console.WriteLine("Enter a number to produce a vector by a number:");
-            var numericInput = ReadInputDoubleAndVerificate();
-            Console.WriteLine($"{vectorFirst.ToString()} * {numericInput.ToString()} = {(vectorFirst * numericInput).ToString()}");
-            Console.WriteLine($"{vectorSecond.ToString()} * {numericInput.ToString()} = {(vectorSecond * numericInput).ToString()}");
+            CallSimpleActionsWithVectors(vectors);
+            CallVectorMultiplication(vectors);
+            CallMultiplyVectorsByNumber(vectors);
+        }
 
-            Console.WriteLine($"{numericInput.ToString()} * {vectorFirst.ToString()} = {(numericInput * vectorFirst).ToString()}");
-            Console.WriteLine($"{numericInput.ToString()} * {vectorSecond.ToString()} = {(numericInput * vectorSecond).ToString()}");
+        private void CallCompareVectors(Vector[] vectors)
+        {
+            var equalityResult = (vectors[0] == vectors[1]);
+            var inequalityResult = vectors[0] != vectors[1];
+            _formattedOutput.ShowCompareVectorsResults(vectors, equalityResult, inequalityResult);
         }
 
-        private static void ShowVectorsAndActions(Vector vectorFirst, Vector vectorSecond)
+        private void CallAngleBetweenVectors(Vector[] vectors)
         {
-            Console.WriteLine("Your vectors:");
-            Console.WriteLine($"1 vector: {vectorFirst.ToString()}");
-            Console.WriteLine($"2 vector: {vectorSecond.ToString()}");
-
-            Console.WriteLine("Actions with vectors:");
-            Console.WriteLine($"{vectorFirst.ToString()} + {vectorSecond.ToString()} = {(vectorFirst + vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} - {vectorSecond.ToString()} = {(vectorFirst - vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} * {vectorSecond.ToString()} = {(vectorFirst * vectorSecond).ToString()}");
+            var angle = VectorHelper.CalculateAngle(vectors[0], vectors[1]);
+            _formattedOutput.ShowAngleBetweenVectorsResult(vectors, angle);
         }
 
-        private Vector EnterUserData(string orderByVectors)
+        private void CallSimpleActionsWithVectors(Vector[] vectors)
         {
-            Console.WriteLine($"Enter the coordinates of the {orderByVectors} three-dimensional vector separated by a space.");
-            var userInput = ReadInputCoordsAndVerificate();
-            return ParseUserInputAndCreateVector(userInput);
+            var sumResult = vectors[0] + vectors[1];
+            var differenceResult = vectors[0] - vectors[1];
+
+            _formattedOutput.ShowSimpleActionsWithVectorsResults(vectors, sumResult, differenceResult);
         }
 
-        private void WriteErrorMessage()
+        private void CallScalarMultiplication(Vector[] vectors)
```

Comment:
Where do you use this method?

Reply:
I fixed. This method really was not called.